README.md

```diff
@@ -190,11 +211,11 @@ Install
           confluent-kafka  using the command in the "Install from
           source from PyPi" section below.
 
-**Install self-contained binary wheels for OSX and Linux from PyPi:**
+**Install self-contained binary wheels for OSX and Linux from PyPi**
```

Comment:
And Windows, so not really sure we need to spell out the platforms since we provide wheels for all relevant ones.

Reply:
sounds good, will make the change in this PR